docs/se/grpc/client.adoc

```diff
@@ -404,41 +404,40 @@ The service name and the method name ("Lower") are specified.
 <2> Set the request type of the method to be `StringMessage` (since the `Lower` method takes `StringMessage` as a parameter).
 <3> Set the response type of the method to be `StringMessage` (since the `Lower` method returns a `StringMessage` as a parameter).
 <4> Build the `ClientMethodDescriptor`. Note that the return value is a `ClientMethodDescriptor` that contains
-the correct Marshallers for the request & response types.
+the correct Marshaller for the request & response types.
```

Comment:
If "Marshaller" here is a class or interface name, enclose it in backticks (that is, `Marshaller`). If it is not a class or interface name and is instead a general noun, use lower case (marshaller).